sources/Launcher/DYNMarginCalculationLauncher.cpp

```diff
@@ -881,30 +881,44 @@ MarginCalculationLauncher::createOutputs(std::map<std::string, std::string>& map
   for (std::vector<LoadIncreaseResult>::const_iterator loadIncreaseResultIt = results_.cbegin();
         loadIncreaseResultIt != results_.cend(); ++loadIncreaseResultIt) {
     const double loadLevel = loadIncreaseResultIt->getResult().getVariation();
-    const std::vector<SimulationResult>& allScenariosResults = loadIncreaseResultIt->getScenariosResults();
-    size_t numberOfSuccessfulScenarios = 0;
-    for (const SimulationResult& scenarioResult : allScenariosResults) {
-      const std::string& scenarioId = scenarioResult.getScenarioId();
-      if (scenarioResult.getSuccess()) {
-        ++numberOfSuccessfulScenarios;
-        const std::map<std::string, SimulationResult>::const_iterator itBest = bestResults.find(scenarioId);
-        if (itBest == bestResults.end() || (loadLevel > itBest->second.getVariation()))
-          bestResults[scenarioId] = scenarioResult;
+    if (loadIncreaseResultIt->getScenariosResults().empty()) {
+      if (loadIncreaseResultIt->getResult().getSuccess()) {
+        const std::map<std::string, SimulationResult>::const_iterator loadIncreaseBestIt = bestResults.find(LOAD_INCREASE);
+        if (loadIncreaseBestIt == bestResults.end() || loadLevel > loadIncreaseBestIt->second.getVariation()) {
+          bestResults[LOAD_INCREASE] = loadIncreaseResultIt->getResult();
```

Comment:
a load increase with no successful scenario cannot be considered better than a successful load increase with successful scenarios

Reply:
in fact this should not happen: if the load increase is succesful then it should have some scenario results